modules/common/src/main/java/unnamed/mmo/util/FutureUtil.java

```diff
@@ -0,0 +1,15 @@
+package unnamed.mmo.util;
+
+import org.jetbrains.annotations.Contract;
+
+public class FutureUtil {
+
+    @Contract("_ -> null")
+    @SuppressWarnings("TypeParameterUnusedInFormals")
+    public static <T> T handleException(Throwable throwable) {
+        //todo log to sentry or something
```

Comment:
Yeah, its just a generic function to be used in handling exceptions in futures where they should not happen.

Errorprone will (rightfully) get angry if you do not add an exceptionally block to CFs, because exceptions in that case will be silenced. For example, [here](https://github.com/minestommmo/JointMMO/blob/feat/chat/v1/modules/chat/src/main/java/unnamed/mmo/chat/ChatManager.java#L63) there is nothing to be done if there is an error, just log it (sentry is a tool for collecting errors that may be used in the future, but also just going to a file is fine for now)